relay-general/src/store/regexes.rs

```diff
@@ -68,7 +68,9 @@ pub static SQL_NORMALIZER_REGEX: Lazy<Regex> = Lazy::new(|| {
     ((?-x)(?P<single_quoted_strs>('(?:[^']|'')*?(?:\\'.*|[^']')))) |
     # Don't capture double-quoted strings (eg used for identifiers in PostgreSQL).
     # Capture numbers.
-    ((?-x)(?P<number>(-?\b(?:[0-9]+\.)?[0-9]+(?:[eE][+-]?[0-9]+)?\b)))
+    ((?-x)(?P<number>(-?\b(?:[0-9]+\.)?[0-9]+(?:[eE][+-]?[0-9]+)?\b))) |
+    # Capture booleans (as full tokens, not as substrings of other tokens).
+    ((?-x)(?P<bool>(\b(?:true|false)\b)))
```

Comment:
Is this case sensitive? Because

`select tRuE or FaLsE` is valid postgreSQL.

Reply:
Yes, the `i` modifier in the first line of the RegEx is for case insensitive. I added a test to ensure this in [a17ce05](https://github.com/getsentry/relay/pull/2121/commits/a17ce057c242072709f4b059a5bac555610ef936).